layouts/partials/search.html

```diff
@@ -2,15 +2,15 @@
 		<div class="searchbox">
 			<input data-search-input id="search-by" type="text" placeholder="{{T "Search-placeholder"}}">
 		</div>
-		<script type="text/javascript" src="{{"js/lunr.min.js" | relURL}}"></script>
-		<script type="text/javascript" src="{{"js/auto-complete.js" | relURL}}"></script>
-		<link href="{{"css/auto-complete.css" | relURL}}" rel="stylesheet">
+		<script type="text/javascript" src="{{ .Site.Data.Solo.DocsVersion }}{{"js/lunr.min.js" | relURL}}"></script>
```

Comment:
does this mean `.Site.Data.Solo.DocsVersion` ends in a `/`? what does a typical docs version look like?

Reply:
yes, I will document this in the readme
It will either be "gloo/some-version/" or an empty string ""